hack/.vendor-helpers.sh

```diff
@@ -112,7 +112,7 @@ clean() {
 	local prune=( $(find vendor -depth -type d -not '(' "${findArgs[@]}" ')') )
 	unset IFS
 	for dir in "${prune[@]}"; do
-		find "$dir" -maxdepth 1 -not -type d -exec rm -v -f '{}' +
+		find "$dir" -maxdepth 1 -not -type d -not -name "LICENSE*" -exec rm -v -f '{}' +
```

Comment:
Would you mind switching this to single quotes?  Also, do we want to include `COPYING*` as well, just in case? (not sure if any of our deps have one)